Resources/Prototypes/_RMC14/Entities/Objects/Misc/bushes.yml

```diff
@@ -858,6 +858,10 @@
   - type: Sprite
     sprite: _RMC14/Objects/Flora/flora_plantalien.rsi
     state: alienplant1
+  - type: PointLight
+    castShadows: false
+    radius: 2
+    color: "#F1d450"
```

Comment:
13 does have a system for colored lights so I assume the choice to use white is intentional there
Personally I think it looks better as white anyways specially over water
```suggestion
```

Reply:
Mmm alright